Updated ES-DE Frontend to 3.2.0

leonstyhre opened the pull request:
Updated ES-DE Frontend to 3.2.0

Begasus commented:
Could you squash the commits?

leonstyhre commented:
Hi, how can I squash the commits? And I guess since I already pushed an update I can't run the following as proposed?
git rebase origin master

Begasus commented:
Merged now, for future references it's best to work with branches instead of using the master branch (been there myself, so no worries) :)
A small self guide for squashing commit I've written is at: https://github.com/Begasus/haikuports/wiki/Using-git-commands#squashing-git-commits

leonstyhre commented:
@Begasus Thanks a lot for this! I have added a comment to my release checklist to make future updates in a separate branch and to squash the commits, so hopefully it's all correct when I make the next update!

And like mentioned by @OscarL yes the `es_de-3.1.1.recipe` file should be removed as I replaced it with  `es_de-3.2.0.recipe`

leonstyhre commented:
@Begasus I still only see ES-DE 3.1.1 in HaikuDepot, so it seems like the merge didn't work correctly?

leonstyhre commented:
Thanks for the reply, I'll wait a few more days to see whether the 3.2.0 release shows up in HaikuDepot!

leonstyhre commented:
Actually, I just realized this message in the page you linked to:

some packages already exist at /var/buildmaster/haikuports/packages, revision bump required

It states it skipped ES-DE, not sure what's that about since it's for sure a new package version?

leonstyhre commented:
Thanks a lot for your reply, I can confirm that it's all good now, I got ES-DE updated to 3.2.0 in R5. And like discussed above, for the next release I'll try to do it correctly by using a separate branch and squashing the commits!
